design-system/src/_shared_customElements.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Workaround import for cross-dependencies between stories.
+ *
+ * CustomElement registration for components shared between stories needs to be done exactly once.
+ */
+
+import NHButton from "./button";
+import NHCard from "./card";
+import NHPageHeaderCard from './page-header-card'
+
+let loaded = false
+if (!loaded) {
```

Comment:
actually this does nothing lol so I'm guessing it's the ESModule resolution behaviour that actually fixed this? If you get any crashes when toggling between different components, this is why...

Reply:
This likely isn't needed due to the Scoped Components Registry handling the registry instead of the global registry. However, I haven't looked through how the components are being used by the storybook, so it's just a hunch based off how the scoped element registry works.

Reply:
Ah, I see how the components are trying to be rendered. I don't think this approach will work properly.

I think we need to define a custom component that uses the scoped component and then make a call to render that component. It would be much like I do in the test for the table, just creating a simple wrapper that extends the `ScopedRegistryHost(LitElement)` and globally registers itself: https://github.com/adaburrows/table-web-component/blob/main/test/table-test-simple.ts

We can probably create a template that can be automatically created in the build process that can be used in the stories. Or maybe a higher order function would suffice to create the proper renderer.

It's also possible the CSS isn't always being included properly, since it really should be built into a module that exports a Lit CSSResult template literal.

Reply:
Logged in #7.